packages/twenty-front/src/modules/object-record/record-field/meta-types/display/components/ChipFieldDisplay.tsx

```diff
@@ -1,23 +1,21 @@
-import { EntityChip } from 'twenty-ui';
-
+import { RecordChip } from '@/object-record/components/RecordChip';
 import { useChipFieldDisplay } from '@/object-record/record-field/meta-types/hooks/useChipFieldDisplay';
+import { RecordIndexRecordChip } from '@/object-record/record-index/components/RecordIndexRecordChip';
 
 export const ChipFieldDisplay = () => {
-  const { recordValue, generateRecordChipData } = useChipFieldDisplay();
+  const { recordValue, objectNameSingular, isLabelIdentifier } =
+    useChipFieldDisplay();
 
   if (!recordValue) {
     return null;
   }
 
-  const recordChipData = generateRecordChipData(recordValue);
-
-  return (
-    <EntityChip
-      entityId={recordValue.id}
-      name={recordChipData.name as any}
-      avatarType={recordChipData.avatarType}
-      avatarUrl={recordChipData.avatarUrl ?? ''}
-      linkToEntity={recordChipData.linkToShowPage}
+  return isLabelIdentifier ? (
+    <RecordIndexRecordChip
```

Comment:
Not sure about this naming RecordIndexRecordChip. should be RecordIdentifierChip?